doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,417 @@
+==================================
+Single Particle Sedimentation
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters used in the DEM and CFD-DEM solvers.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a single particle sedimentation in water
+- Use of ``qcm`` void fraction scheme with ``gauss-lobatto`` quadrature
+- Python post-processing script using `PyVista <https://docs.pyvista.org/>`_
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/single-particle-sedimentation``).
+
+- Parameter file for initial particles generation: ``initial-particle.prm``
+- Parameter file for CFD-DEM simulation of the Boycott effect: ``single-particle-sedimentation.prm``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the sedimentation of a single particle in a water tank due to gravity. As other unresolved CFD-DEM applications and examples, we start it but introducing the particle in our domain by running a DEM simulation (using ``lethe-particles``). The DEM simulation generates checkpoint files that will be used by our unresolved CFD-DEM application (``lethe-fluid-particles``) to determine the initial position of the particle.
```

Comment:
I would replace "other" by "all", since this is the first example that appears in the list of unresolved CFD-DEM:

```suggestion
This example simulates the sedimentation of a single particle in a water tank due to gravity. As all unresolved CFD-DEM applications and examples, we start it by introducing the particle in our domain by running a DEM simulation (using ``lethe-particles``). The DEM simulation generates checkpoint files that will be used by our unresolved CFD-DEM application (``lethe-fluid-particles``) to determine the initial position of the particle.
```